payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AddressSpec.kt

```diff
@@ -1,26 +1,47 @@
 package com.stripe.android.ui.core.elements
 
+import android.os.Parcelable
 import androidx.annotation.RestrictTo
 import com.stripe.android.ui.core.R
 import com.stripe.android.ui.core.address.AddressFieldElementRepository
 import kotlinx.parcelize.Parcelize
+import kotlinx.serialization.Serializable
 
+@Serializable
 @Parcelize
+@Suppress("EnumEntryName")
+enum class DisplayField : Parcelable {
+    country
+}
+
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP_PREFIX)
+@Serializable
+@Parcelize
```

Comment:
Why does this need to be both Serializable and Parcelable? Can we make it implement only Parcelable?

Reply:
See other PR: https://github.com/stripe/stripe-android/pull/5082